ui/styl/fonts.styl

```diff
@@ -0,0 +1,22 @@
+// partially copied from https://github.com/code42day/stylus-font-face
```

Comment:
move this file under modules... actually, I would create a 'font.styl' under modules which only includes the font-face() and file() function, and then move the actual definitions to 'typography.styl'.

The idea behind the current organization was for `modules/` to contain only things with no actual output (like functions), while files in the `partials/` directory generate css for different pieces of the layout. I based this on some blog posts, which you can probably find in my commit messages.